main.tf

```diff
@@ -297,7 +297,7 @@ resource "aws_codebuild_project" "this" {
     compute_type = "BUILD_GENERAL1_SMALL"
 
     // https://docs.aws.amazon.com/codebuild/latest/userguide/build-env-ref-available.html
-    image           = "aws/codebuild/docker:18.09.0"
+    image           = "aws/codebuild/ubuntu-base:14.04"
```

Comment:
Hmmpf, `14.04` is quite ancient! 🌋 Is there a newer image? Maybe `18.04`? 🤔

Reply:
That's the newest they do support 👎 

https://docs.aws.amazon.com/codebuild/latest/userguide/build-env-ref-available.html

Reply:
🤮